storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,17 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
+  creation_tx HEX64 NOT NULL,
+  bytecode BYTEA,
+  abi JSONB,
+  verified BOOLEAN NOT NULL,
+  PRIMARY KEY (runtime, contract_address)
+);
```

Comment:
Was there a discussion on what fields to store for verified contracts? What data will be useful/needed on frontend.

In addition to ABI and bytecode, the following come to mind (by looking at some other explorers):
- compiler version used do compile the contracts (e.g. `0.8.18+commit.87f61d96`)
- optimization (and other) settings used when compiling the contracts
- source code


Not really an action item for this PR because all verified fields will be empty in any case, just seems like a good place for the discussion.